lib/rubocop/cop/lint/return_in_void_context.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for the use of a return with a value in a context
+      # where it the value will be ignored. (initialize and setter methods)
+      #
+      # @example
+      #
+      #   # bad
```

Comment:
I guess the cause is not 3 spaces, because documentations of other cops have 3 space(e.g. https://github.com/bbatsov/rubocop/blob/f6eef8a2cf46b05b2088377959da975c31fa4efc/lib/rubocop/cop/style/accessor_method_name.rb ).

I can fix the broken documentation by adding a `@example` in #4468 .